helm/k8s-dns-node-cache-app/templates/configmap.yaml

```diff
@@ -13,8 +13,7 @@ data:
                 rcode NOERROR
         }
         cache {
-                success 9984 {{ .Values.configmap.cache.success }}
```

Comment:
why did we remove these? afaik they come from upstream chart, there might be a reason why

Reply:
After some digging I found this commit here explaing some issues for when `denial` is too high. I adjusted my PR to keep the 2 TTLs for each cache.